samples/interactive-tutorials/test/import-user-events-big-query.test.js

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const path = require('path');
+const cp = require('child_process');
+const {before, after, describe, it} = require('mocha');
+const {assert, expect} = require('chai');
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+const cwd = path.join(__dirname, '..');
+const utils = require('../setup/setup-cleanup');
+
+describe('Import user events from big query table', () => {
+  let stdout;
+  const dataset = `user_events_${Math.round(Date.now() / 1000)}`;
+  const table = 'events';
+  const schema = 'interactive-tutorials/resources/events_schema.json';
+  const sourceFile = 'interactive-tutorials/resources/user_events.json';
+
+  before(async () => {
```

Comment:
In order to avoid using env variables, especially for running tests locally, we can require a `projectId` variable, and in the test, instantiate it like so: 

```
const client = new UserEventServiceClient();
describe('test', () => {
  let projectId;

  before(async () => {
    projectId = await client.getProjectId();
  });
```